torch/lib/THD/base/data_channels/DataChannelMPI.cpp

```diff
@@ -96,8 +96,8 @@ bool DataChannelMPI::init() {
   int* provided = NULL;
   MPI_Init_thread(NULL, NULL, MPI_THREAD_MULTIPLE, provided);
   if (*provided != MPI_THREAD_MULTIPLE) {
-    std::cerr << "MPI implementation does not support multiple threads."
-              << "Using same data channel in multiple thread can result in"
+    std::cerr << "WARNING: MPI implementation does not support multiple threads. "
+              << "Using same data channel in multiple thread can result in "
```

Comment:
Can you change the message as I said? The user doesn't need to be aware of what the DataChannel is, so it's best to make the message simple yet still convey the information that it might give incorrect results.